app/src/main.ts

```diff
@@ -179,83 +179,107 @@ if (shouldQuit) {
     }
   });
 
-  app.on('ready', () => {
-    mainWindow = createMainWindow(appArgs, app.quit.bind(this), setDockBadge);
-    createTrayIcon(appArgs, mainWindow);
-
-    // Register global shortcuts
-    if (appArgs.globalShortcuts) {
-      appArgs.globalShortcuts.forEach((shortcut) => {
-        globalShortcut.register(shortcut.key, () => {
-          shortcut.inputEvents.forEach((inputEvent) => {
-            mainWindow.webContents.sendInputEvent(inputEvent);
-          });
+  if (appArgs.widevine !== undefined) {
+    // @ts-ignore
```

Comment:
Here and below: I like to comment why `@ts-ignore` is used when it is. Pls inline comment each of these. TSC is fine with it. For example,

```suggestion
    // @ts-ignore because blah blah blah. Alternative is to foo, but we prefer bar.
```